python/prima/_bounds.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from scipy.optimize import Bounds
+
+def process_bounds(bounds, lenx0):
+    '''
+    bounds can either be an object with the properties lb and ub, or a list of tuples
```

Comment:
Let us write "Bounds ..." 

Even in comments, I think we should write sentenses as if we are writing a formal paper or report.  People will judge the quality of the code by looking at the comments. Another particular reason is that the comments will be shown in the documentaion "as is" if we use documentation tools such as Doxygen.